Rakefile

```diff
@@ -80,7 +80,7 @@ class RepositoryListUpdater
             <tr>
               <td><a href="#{owner_url(fqdn, owner)}">#{h(owner)}</a></td>
               <td>#{repository_column}</td>
-              <td>#{h(to)}</td>
+              <td><a href="mailto:#{h(to)}?cc=null@commit-email.info&amp;subject=Subscribe&amp;body=subscribe">#{h(to)}</a></td>
```

Comment:
Could you extract this `mailto:` URL generate code as a method like `owner_url` and `repository_url`?

Reply:
Thank you for your advise.
It will make the code very readable.
I will fix it.

Reply:
I have fixed this.